docs/use-cases/navigate.md

```diff
@@ -1,8 +1,54 @@
 ---
-title: "Teleoperate and navigate rovers"
+title: "Teleoperate and navigate with rovers"
 linkTitle: "Teleoperate and navigate with rovers"
-weight: 60
+weight: 50
 type: "docs"
-layout: "empty"
-canonical: "tutorials/services/navigate-with-rover-base/"
+tags: ["navigation", "motion", "services", "base", "base remote control"]
+no_list: true
+description: "Use Viam to teleoperate and navigate rover bases."
+image: "/services/icons/navigation.svg"
+imageAlt: "Navigation"
+images: ["/services/icons/navigation.svg"]
 ---
+
+If you have a rover base, you can teleoperate and navigate it with Viam.
+Remotely control your base after configuring your machine on the app's **Control** tab, and set up autonomous navigation with the [navigation service](/mobility/navigation/).
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/use-cases/base-control.png" class="fill alignright" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base control card">}}
+      <b>1. Teleoperate</b>
+      <p>Remotely control your rover base through the <a href="https://app.viam.com">Viam app.</a> Create an account and add a machine, <a href="/get-started/installation/">install viam-server</a>, and <a href="/components/base/">configure your base</a>. Then, go to the <b>Control</b> tab and access a remote control card for your base, with an interface for controlling speed, direction, and power.</p>
```

Comment:
```suggestion
      <p>You can remotely control your rover from anywhere through the <a href="https://app.viam.com">Viam app.</a> Create an account and add a machine, <a href="/get-started/installation/">install viam-server</a>, and <a href="/components/base/">configure your rover base</a>. Then, go to the <b>Control</b> tab and access a remote control card for your base, with an interface for controlling speed, direction, and power. You can also view live feeds from any cameras you configure.</p>
```
With the current imperative mood first sentence, it sounds like someone should start remotely controlling it already, and then make an account etc.

Reply:
agreed. Was going to suggest something similar